### ISSUE TITLE
pre_tweet_hook

### ISSUE THREAD

**Issue by karlh626** [mdom/txtnish#5]
I dont see a pre_tweet_hook in the readme - is that function available yet?

**Comment by mdom** [comment]
> I dont see a pre_tweet_hook in the readme - is that function available yet?
> 
No, it isn't. I never had a use case for that hook. Do you have one in mind?

Cheers, Mario

**Comment by karlh626** [comment]
I have multiple laptops with twtxt installed each with their own local twtxt.txt file.  I then use the pre_tweet_hook to first scp the existing copy from the webserver so that my local instance is the most recent.  This is to make sure that my local copy is up to date first in case I updated the twtxt.txt from another machine recently.  After this pre_tweet_hook updates my local copy, the new tweet gets appended, then the post_tweet_hook uploads the new twtxt.txt to the webserver.

If there is a better way to accomplish this, I am also interested as this was the only way I came up with to handle having multiple sources.
